docs/source/interfaces/isis_sans/Sum Runs.rst

```diff
@@ -0,0 +1,138 @@
+| **Remove**                | Removes an entry from the summation table. Note, this does not delete the file itself,  |
+|                           | it just removes it from the list of files to be summed.                                 |
++---------------------------+-----------------------------------------------------------------------------------------+
+| **Remove All**            | Removes all entries from the summation table. As above, this will only remove the       |
+|                           | entries from the table, not the files themselves.                                       |
++---------------------------+-----------------------------------------------------------------------------------------+
+
+Histogram Binning
+^^^^^^^^^^^^^^^^^
+
+This panel allows the user to specify the binning parameters to be applied
+when summing event data. There are three different ways to add files containing
+event data [#no-effect-when-non-event]_.
+
+Custom Binning
+""""""""""""""
+
+If this option is chosen a line edit field becomes available which the user
+can use to set the preferred binning boundaries. The format of this input is
+identical to the format required by
+the :doc:`Rebin Algorithm </algorithms/Rebin-v1>`.
+
+
+Use Binning From Monitors
+"""""""""""""""""""""""""
+
+If this option is chosen the binning is taken from the monitors.
+
+Save As Event Data
+""""""""""""""""""
+
+If this option is chosen, the output file will contain event data.
+The output is not an event workspace but rather a group workspace, which
+contains two child event workspaces, one for the added event data and one
+for the added monitor data.
+
+With **'Overlay Event Workspaces' Disabled** the event data from the files
+is added using the event the :doc:`Plus Algorithm <../algorithms/Plus-v1>`.
+Timestamps of the events and of the logs are not
+changed as indicated in the image below.
+
+.. figure:: /images/ISISSansInterface/sum_runs_add_no_overlay.png
+   :align: center
+   :width: 600px
+
+   Simple addition of event data
+
+With **'Overlay Event Workspaces' Enabled** and **no Additional Time Shifts**
+specified, the event data of the different files is shifted on top of each
+other.
+
+In the case of two workspaces the time difference between them is determined
+by the difference between their first entry in the proton charge log.
+This time difference is then applied to all timestamps of the second workspace.
+
+The second workspace is essentially laid on the first. The same principle
+applies if more than two workspaces are involved as this is a pairwise
+operation. The working principle is illustrated below:
+
+.. figure:: /images/ISISSansInterface/sum_runs_add_overlay.png
+   :align: center
+   :width: 600px
+
+   Adding two workspaces by overlaying them
+
+Note that the underlying mechanism for time shifting is provided by the
+:doc:`ChangeTimeZero Algorithm <../algorithms/ChangeTimeZero-v1>`.
```

Comment:
:ref:`algm-ChangeTimeZero`

Reply:
there may be others, but these were failing to link up and causing sphinx warnings